Datadog/Example/IntegrationTestFixtures/SendTracesFixtureViewController.swift

```diff
@@ -55,6 +56,17 @@ internal class SendTracesFixtureViewController: UIViewController {
         super.viewDidAppear(animated)
 
         viewAppearingSpan.finish()
+
+        // Step #2: Auto Instrumentation
+        // Send two requests which will be automatically traced as tracing auto-instrumentation is enabled
+        let tracedHost = currentAppConfig().sourceEndpoint.absoluteString
+        let url = URL(string: "\(tracedHost)/dataTaskWithURL")!
+        var request = URLRequest(url: URL(string: "\(tracedHost)/dataTaskWithRequest")!)
```

Comment:
This is OK for running the Example app in `UITests`. But when launching it on Simulator or device to seed data to Traces Explorer, this results with hitting the `404` resource: `https://app.datadoghq.com//dataTaskWithURL`. This misaligns the behaviour between tests vs regular launch:
* Tests: `200` OK for `GET`,  `200` OK for `POST`,
* Regular Launch: `404` error for `GET`, `404` error for `POST`.

Because of this, we don't seed enough trace data to Traces Explorer for lookup and reference, as it's done for `"view appearing"` trace. The `"view appearing"` trace contains 3 spans, from which 2 are OK and 1 is error.  **By "not enough trace data" I mean: we don't even know how both, the "ok" and "error" spans send from tracing interceptor will look in Traces Explorer** and this is IMO critical to implement.

I think we can easily fix it by differentiating the `GET` and `POST` urls in the `Example` app:
* for tests, pass two distinct Python server `.recordingURL` (by obtaining two server sessions),
* for regular launch, just configure two URLs in `ExampleAppConfig`.

As this will still produce two `200s` in tests (vs ` GET → 200` and `POST → 4xx` in regular launch), we can further improve it (in separate JIRA), so that Python server can return `4xx` for predefined POST request.

WDYT @buranmert ?

Reply:
summary: basically that reads we should send one OK and one ERROR span in `ExampleApp`; duplicate this in tests when mock server can return failure as well.

we can make 2 requests: `www.good-url.com` and `www.good-url.com/bad-path`, that result 1 OK and 1 ERROR in App; test always has 2 OKs for now.

Reply:
as `MockServer` doesn't accept `GET` requests, i went with a sub-optimal solution/workaround

`AppConfig` has `arbitraryNetworkURL/Request`, `ExampleAppConfig` and `UITestsConfig` provide different values